### PR TITLE
Update formal-review-process.md

### PR DESCRIPTION
**Pull request by lynn915**

### CODE REVIEW

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 1]
```diff
@@ -12,13 +12,15 @@ Before each review cycle, Managers read the performance review chapter in High O
...
+All FT staff partcipating in the review cycle will receive a notification via email as well as a task in your Culture Amp inbox inviting you to nominate peers for feedback. To get started, click **complete nominations** from the task and search for the peers you'd like to nominate. (Note that Culture Amp pulls its information directly from BambooHR, so legal names will appear in Culture Amp. If you can't find a peer, please reach out to hr@mattermost.com). Confirm nominations by selecting **send nominations**. We recommend selecting 3-6 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations and deciding who to nominate is available [here](https://support.cultureamp.com/hc/en-us/articles/360016973880-Performance-Review-Cycle-Guide-for-Managers-and-Employees).
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:15]:
```suggestion
All full-time staff participating in the review cycle will receive a notification via email as well as a task in Culture Amp inbox inviting them to nominate peers for feedback. To get started, select **complete nominations** from the task and search for the peers you'd like to nominate. (Note that Culture Amp pulls its information directly from BambooHR, so legal names will appear in Culture Amp. If you can't find a peer, please reach out to hr@mattermost.com for assistance). Confirm nominations by selecting **send nominations**. We recommend selecting 3-6 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations and deciding who to nominate is available [here](https://support.cultureamp.com/hc/en-us/articles/360016973880-Performance-Review-Cycle-Guide-for-Managers-and-Employees). 
```

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 2]
```diff
@@ -12,13 +12,15 @@ Before each review cycle, Managers read the performance review chapter in High O
...
+
+Once your direct report has completed their nominations, Managers will receive an email and a task in your Culture Amp inbox. You are able to either (a) accept your direct report's nominations and immediately send feedback requests to their peer selection, or (b) edit their nominations or add new peers for additional feedback. Depending on the role, PT Staff and community members may be solicited to provide feedback in addition to the peer panel selected in Culture Amp.To complete, select **schedule requests**. Your requests will be scheduled to be sent when the feedback stage starts. Managers may return to this page to send more requests up until the end of the feedback stage. More information for managers on selecting peer panels is available [here](https://academy.cultureamp.com/hc/en-us/articles/360003789699-Request-team-feedback).
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:17]:
```suggestion
Once direct reports have completed their nominations, Managers will receive an email and a task in their Culture Amp inbox. Managers can either (a) accept their direct report's nominations and immediately send feedback requests to their peer selection, or (b) edit their nominations or add new peers for additional feedback. Depending on the role, part-time Staff and Community Members may be solicited to provide feedback in addition to the peer panel selected in Culture Amp. To complete the nomination process as a Manager, select **schedule requests**. Your requests will be scheduled to be sent when the feedback stage begins. Managers may return to this page to send more requests up until the end of the feedback stage. More information for Managers on selecting peer panels is available [here](https://academy.cultureamp.com/hc/en-us/articles/360003789699-Request-team-feedback).
```

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 3]
```diff
@@ -12,13 +12,15 @@ Before each review cycle, Managers read the performance review chapter in High O
...
 
 Prior to selecting peer panels and writing performance reviews for their direct reports, managers should take the Culture Amp training available [here](https://www.cultureamptraining.com/performance-for-managers).
 
 ## Peer feedback
 
-Managers will select the peer panels in Culture Amp. To get started, log in to Culture Amp and navigate to your **task list** where you'll find requests for peer or upward feedback. On average, teammates should be completing feedback surveys for 3 - 6 of their peers.
+Staff who have received peer feedback requests will receive an email and a task in their Culture Amp inbox. To complete the request, click **give feedback**. Peers may decline the request based on bandwidth (more than 5 peer feedback requests), if they have give feedback in the previous review cycle and feel that their feedback would be duplicative, or if they don't have feedback to share.
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:23]:
```suggestion
Staff who have received peer feedback requests will receive an email and a task in their Culture Amp inbox. To complete the request, select **give feedback**. Peers may decline the request based on bandwidth (with more than 5 peer feedback requests), if they have gave feedback in the previous review cycle and feel that their feedback would be duplicative, or if they don't have feedback to share. 
```

**Review by cwarnermm — COMMENTED** [review]
Great update! Feedback provided inline.

**Review by lynn915 — COMMENTED** [review]
This looks great, @cwarnermm . You can publish this one too. Thanks!

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**